ext_mongodb.php

```diff
@@ -1084,6 +1099,11 @@ public function serialize() : string
 		return serialize( $s );
 	}
 
+	public function jsonSerialize() : mixed
+	{
+		return $this->code;
```

Comment:
I'll note that you're being consistent with libmongoc here (code as string and ignoring the scope). There is no corresponding extended JSON representation: https://docs.mongodb.com/v3.2/reference/mongodb-extended-json/#bson-data-types-and-associated-representations